scripts/release_support/changelog.sh

```diff
@@ -58,8 +58,34 @@ export TZ=$time_zone;ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ
 
 pushd $clone_dir/clone
 
-wget -q $BUILD_LOCATION/$START_BUILD/dev/RELEASE.INFO -O start_build_manifest.txt
-wget -q $BUILD_LOCATION/$TARGET_BUILD/dev/RELEASE.INFO -O target_build_manifest.txt
+if [ -z $BUILD_LOCATION ]; then
+	NOT_DOCKER_IMAGE=$(echo "$START_BUILD"|grep RELEASE.INFO|wc -l)
```

Comment:
@AbhijitPatil1992, What do you mean by variable `NOT_DOCKER_IMAGE`? can we simply say `DOCKER_IMAGE` or some meaningful name which correctly represent what variable contains?

Reply:
@AbhijitPatil1992 Can you please address this comment by @gauravchaudhari02

Reply:
Added variable name as a CHECK_INPUT_VARIABLE_STATUS

Reply:
@gauravchaudhari02  can you please review now. I have replaced NOT_DOCKER_IMAGE to CHECK_INPUT_VARIABLE_STATUS. Because here we are checking if input variable is docker image or not.